content/04.results.md

```diff
@@ -6,41 +6,41 @@ Panel A, left, depicts an example news article and the type of data extracted fr
 
 
 
 ### Creation of an Annotated News Dataset
 
 We have analyzed the text and citations of 22,094 news-related articles hosted on "www.nature.com".
-The articles span 15 years from 2005-2020, and seven article types: Career Column, Career Feature, News, News and Views, News Feature, Technology Feature, and Toolbox.
-The text and citations were then uniformly processed as depicted in Figure @fig:fig1 a.
+The articles span 15 years from 2005 to 2020, and seven article types: Career Column, Career Feature, News, News and Views, News Feature, Technology Feature, and Toolbox.
+The text and citations were then uniformly processed as depicted in Figure {@fig:fig1}a.
 We scraped the text using the web-crawling framework Scrapy [@https://scrapy.org/], processed, and run through the coreNLP pipeline ([Methods]).
 From the coreNLP pipeline results, we wanted to identify two distinct data types to analyze, country mentions and quotes with their associated speakers.
 To identify country mentions, we used the following named entities as possible mentions: "organizations", "countries", "states or provinces".
 We then mapped the named entity to a country prediction using OpenStreetMap [@https://www.openstreetmap.org].
 To identify quotes and speakers, we used the coreNLP quote extraction and attribution annotator.
 We performed name normalization ([Methods]) to identify the speaker’s full name for gender and name origin prediction.
 We scraped the citations using an independent scraper to the text scraper.
-All identified DOI's were queried using the Springer API to attain author names, positions, and affiliations.
+All identified DOI's were queried using the _Springer_ API to attain author names, positions, and affiliations.
 
 
-We used the author names, positions, and affiliations of Research articles and Letters published by Nature to perform comparative analyses.
-Since the news articles of current science, the quoted speakers, mentioned countries, and cited authors should have a similar demographic makeup to contemporaneous Nature authorship.
+We used the author names, positions, and affiliations of Research articles and Letters published by _Nature_ to perform comparative analyses.
+Since the news articles of current science, the quoted speakers, mentioned countries, and cited authors should have a similar demographic makeup to contemporaneous _Nature_ authorship.
 The author metadata of research-related articles was analyzed over the same period as the news-related articles and totaled 13,414 articles.
-Since the demographic makeup of authors published in Nature may not represent overall science authorship, we also analyzed 36,000 randomly selected articles from English language journals over the same time.
+Since the demographic makeup of authors published in _Nature_ may not represent overall science authorship, we also analyzed 36,000 randomly selected _Springer_ published articles from English language journals over the same time.
```

Comment:
```suggestion
Since the demographic makeup of authors published in _Nature_ may not represent overall science authorship, we also analyzed 36,000 randomly selected _Springer_-published articles from English language journals over the same time.
```